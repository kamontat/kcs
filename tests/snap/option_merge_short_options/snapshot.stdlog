[DBG] init.options         reuse definitions from cache file '$TMPDIR/kcs/libs-options/options-definitions.txt'
[DBG] hook-init.options    options definition: |-l|:LAND:NV::;|-f|--flower|:FLOWER:NV::;|-s|--sky|:SKY:RS::;|-w|--water|:WATER:OS::;|-r|--rock|:ROCK:OS:paper:;|-e|--ex|--extra|:EXTRA:NV:paper:;|--q|:QUE:NV:paper:;
[DBG] hook-init.options    parsing short options '-l'
[DBG] checker.options      option '-l' is no argument type
[DBG] export.options       assign argument of NV to 'true'
[DBG] export.options       export '_KCS_OPT_LAND_VALUE'='true'
[DBG] hook-init.options    parsing short options '-f'
[DBG] checker.options      option '-f' is no argument type
[DBG] export.options       assign argument of NV to 'true'
[DBG] export.options       export '_KCS_OPT_FLOWER_VALUE'='true'
