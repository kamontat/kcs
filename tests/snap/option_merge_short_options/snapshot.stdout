Direct Arguments: 1 [-lf]
Parsed Arguments: 0 []
Land: true
Flower: true
